itest/rust/src/register_tests/func_test.rs

```diff
@@ -148,3 +207,51 @@ impl RefCountedVirtual for GdSelfReference {
         }
     }
 }
+
+#[itest]
+fn cfg_doesnt_interfere_with_valid_method_impls() {
+    // if we re-implement this method but the re-implementation is removed, that should keep the non-removed
```

Comment:
Nitpick: sentence should start with capital letter, and could fit on one line.

Reply:
It seems to surpass 120 characters when kept at a single line, is that ok?

Reply:
Ah, looked shorter 😀 but yes, [120-145 is fine](https://godot-rust.github.io/book/contribute/conventions.html#formatting).

Reply:
Ok!